browser/profiles/brave_profile_manager.cc

```diff
@@ -118,20 +94,16 @@ void BraveProfileManager::DoFinalInitForServices(Profile* profile,
       std::make_unique<brave_content::BraveSharedResourcesDataSource>());
 }
 
-void BraveProfileManager::LaunchTorProcess(Profile* profile) {
-  // TODO(bridiver) - this should all go inside the Tor Service and just
-  // call LaunchTor()
+void BraveProfileManager::OnProfileCreated(Profile* profile,
+                                           bool success,
+                                           bool is_new_profile) {
+  ProfileManager::OnProfileCreated(profile, success, is_new_profile);
+
 #if BUILDFLAG(ENABLE_TOR)
-  tor::TorProfileService* tor_profile_service =
+  // we need to wait until OnProfileCreated to
```

Comment:
nit: As per CC comments should like like a normal sentences

Reply:
not sure what you're asking for here?

Reply:
A comment should start with a capital letter and end with a dot -> `We need to wait ... available.`